<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterFuelFlash</class>
 <widget class="QWidget" name="FilterFuelFlash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="WdtContainer" name="wc" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Partner</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="C5LineEdit" name="lineEdit_3">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>fuel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="C5LineEdit" name="lePartner">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>partner</string>
        </property>
        <property name="lkey" stdset="0">
         <string>partners</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fuel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>352</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="C5LineEdit" name="lineEdit_5">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>move</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="C5LineEdit" name="lineEdit_4">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>partners</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="C5LineEdit" name="leFuel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>fuel</string>
        </property>
        <property name="lkey" stdset="0">
         <string>fuel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbIssue">
        <property name="text">
         <string>Issue date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="C5DateEdit" name="leDate2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateissue2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="C5LineEdit" name="leState">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>move</string>
        </property>
        <property name="lkey" stdset="0">
         <string>move</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="C5DateEdit" name="leDate1">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateissue1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Payment type</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="C5LineEdit" name="lePayment">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>payment</string>
        </property>
        <property name="lkey" stdset="0">
         <string>payment</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="C5LineEdit" name="lineEdit_6">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>payment</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WdtContainer</class>
   <extends>QWidget</extends>
   <header location="global">wdtcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
